src/coreclr/jit/importercalls.cpp

```diff
@@ -3757,6 +3760,14 @@ GenTree* Compiler::impIntrinsic(CORINFO_CLASS_HANDLE    clsHnd,
                 break;
             }
 
+            case NI_System_Type_GetGenericTypeDefinition:
+            {
+                GenTree* type = impStackTop(0).val;
+
+                retNode = impGetGenericTypeDefinition(type);
```

Comment:
it feels weird that we do "stack top" here and "stack pop" inside the function, I think we should do both in the same place

Reply:
I followed the same pattern as `impTypeIsAssignable`, which also does the same. Should I change this?